client/state/selectors/get-sites.js

```diff
@@ -20,7 +20,7 @@ export default createSelector(
 	( state ) => {
 		const primarySiteId = getPrimarySiteId( state );
 		const [ primarySite, sites ] = partition( state.sites.items, { ID: primarySiteId } );
-		return map( concat( primarySite, sites ), site => getSite( state, site.ID ) );
+		return map( concat( primarySite, sortBy( sites, [ 'name', 'URL' ] ) ), site => getSite( state, site.ID ) );
```

Comment:
Aside, this call tree is getting a little deep (and the line long) — maybe we could address that? For instance:

```js
flow(
  sites => sortBy( sites, [ 'name', 'URL' ] ),
  sites => concat( primarySite, sites ),
  sites => map( sites, 'ID' ),
  ids => map( ids, id => getSite( state, id ) )
)( sites )
```

(would look so much nicer with `lodash/fp` or `ramda`, but at least the above makes the chain of transformations clearer, IMO)